docs/AboutUs.md

```diff
@@ -40,12 +40,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Lin Wan Lei
 
-<img src="images/WanLei.png" width="200px">
+<img src="images/linwanlei.png" width="200px">
 
 [[github](http://github.com/linwanleii)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/linwanleii.md)]
 
-* Role: Developer
+* Role: Team Lead, Deliverables and deadlines
 * Responsibilities: Dev Ops + Threading
```

Comment:
Is your responsibilities a dev ops + threading? Perhaps you can put 'Deliverables and deadlines' Responsibilities instead of role

Reply:
Done that thanks